tuning/sft_trainer.py

```diff
@@ -310,13 +312,16 @@ def train(
     # this validation, we just drop the things that aren't part of the SFT Config and build one
     # from our object directly. In the future, we should consider renaming this class and / or
     # not adding things that are not directly used by the trainer instance to it.
+
     transformer_train_arg_fields = [x.name for x in dataclasses.fields(SFTConfig)]
     transformer_kwargs = {
-        k: v
+        "hub_token" if k == "push_to_hub_token" else k: v
```

Comment:
I see the warning in the logs, but how does this `push_to_hub_token` get set? I don't see it as part of the cli command flag?

Reply:
Yeah it's very interesting, because it [defaults to None](https://github.com/huggingface/transformers/blob/main/src/transformers/training_args.py#L1385) in transformers, which is the only time I can find it getting effected, but it still seems to be getting set to _something_, because we are [hitting the warning](https://github.com/huggingface/transformers/blob/main/src/transformers/training_args.py#L2020).